catalog/spatial/ogc/spatial-ogc-urlresourcereader/pom.xml

```diff
@@ -106,7 +114,9 @@
                             ${project.artifactId};blueprint.graceperiod:=true;blueprint.timeout:=604800000
                         </Bundle-SymbolicName>
                         <Embed-Dependency>
-                            platform-util
+                            platform-util,
+                            httpcore,
+                            httpclient
```

Comment:
Hopefully this causes the ci build to work now. I wonder if it would actually make more sense to embed the catalog-core-urlresourcereader instead of embedding its embedded transitive dependencies